lib/SyTest/Identity/Server.pm

```diff
@@ -115,11 +116,57 @@ sub on_request
    }
 }
 
-sub stub_token {
+sub bind_identity {
    my $self = shift;
-   my ( $token, $medium, $invitee_email, $inviter_mxid, $room_id ) = @_;
+   my ( $hs_uribase, $medium, $address, $user, $before_resp ) = @_;
+
+   $self->{bindings}{ join "\0", $medium, $address } = $user->user_id;
+
+   if( !defined $hs_uribase ) {
+      return Future->done( 1 );
+   }
+
+   my %resp;
+   $resp{address} = $address;
+   $resp{medium} = $medium;
+   $resp{mxid} = $user->user_id;
+
+   my $invites = $self->{invites}->{ join "\0", $medium, $address };
+   if( defined $invites ) {
+      foreach my $invite ( @$invites ) {
+         $invite->{mxid} = $user->user_id;
+         $invite->{signed} = {
+            mxid => $user->user_id,
+            token => $invite->{token},
+         };
+         sign_json( $invite->{signed},
+            secret_key => $self->{private_key},
+            origin => $self->name,
+            key_id => "ed25519:0",
+         );
+      }
+      $resp{invites} = $invites;
+   }
 
-   $self->{expected_tokens}{ join "\0", $medium, $invitee_email, $inviter_mxid, $room_id } = $token;
+   sign_json( \%resp,
+      secret_key => $self->{private_key},
+      origin => $self->name,
+      key_id => "ed25519:0",
+   );
+
+   $before_resp->() if defined $before_resp;
+
+   $self->{http_client}->do_request_json(
+      uri => URI->new( "${hs_uribase}/_matrix/federation/v1/3pid/onbind" ),
```

Comment:
Braces aren't required around a simple variable like that; especially as it's not going to get confused with the `/` after it.